code/datums/elements/strippable.dm

```diff
@@ -36,7 +36,6 @@
 /datum/element/strippable/proc/mouse_drop_onto(datum/source, atom/over, mob/user)
 	SIGNAL_HANDLER
 
-	. = COMPONENT_CANCEL_MOUSEDROP_ONTO
```

Comment:
`/datum/element/strippable` (all humans have this element) couldn't be mousedropped because this element was cancelling the mousedrop entirely even if nothing happened

Reply:
Basically, stop doing this! Be explicit about when you want to block and when you don't

Reply:
OHHHHHHHH. 

Yeah my bad. I thought elements that had this this element didn't do anything else